oojs.md

```diff
@@ -7,15 +7,26 @@ Define a `Playlist` class. It should have...
 - An `addSong` (Function) method that adds a song to the `songs` property
 
 ```js
-Your answer goes here
+class Playlist {
+  constructor(title, songs){
+    title: "Great Songs"
+    songs: ["song1", "song2", "song3", "song4"]
+    favorites:
+  }
+  addSong(){
+    this.song ++
+  }
+
+
+}
 ```
 
 ### 2
 
 Create an instance of the `Playlist` class.
 
 ```js
-Your answer goes here
+return Playlist;
```

Comment:
you'd use  the `new` keyword

var antwanasPlaylist = new Playlist("Antwana's PL", ["song1","song2","song3"])